utils/s2n_random.c

```diff
@@ -125,12 +206,12 @@ S2N_RESULT s2n_get_private_random_bytes_used(uint64_t *bytes_used)
     return S2N_RESULT_OK;
 }
 
-S2N_RESULT s2n_get_urandom_data(struct s2n_blob *blob)
+static int s2n_rand_urandom_impl(void *ptr, uint32_t size)
```

Comment:
Why is this file now half int, half S2N_RESULT? Can we keep it consistent?

Reply:
If it's a public callback it'll need to be an `int`. But all of the internal functions should continue to return results.

Reply:
Yup, as far as I can tell, all of the functions that return `int` are callbacks.